pages/checkout-beta/_subscription.html

```diff
@@ -0,0 +1,143 @@
+          </div>
+          <div class="col-xs-6 col-sm-12">
+            <hr class="hr-sm hr-reset visible-sm-block visible-md-block visible-lg-block">
+            <p class="text-muted small text-center margin-bottom-none visible-sm-block visible-md-block visible-lg-block">30-Day Money Back Guarantee</p>
+            <p class="text-muted small text-center margin-bottom-none visible-sm-block visible-md-block visible-lg-block">
+              Annual Contract. Billed monthly.
+            </p>
+            <div style="height: 35px;">
+              <p class="small text-white text-bold margin-top-none text-center" id="deposit-due" style="margin-bottom: 0">Deposit Due<span class="hide-on-tiny"> Today</span>: $500</p>
+            </div>
+            <button
+              type="button"
+              class="btn btn-info btn-block btn-checkout"
+              data-cb-type="checkout"
+              data-cb-plan-id="chargebee-subscription">
+              Checkout
+            </button>
+            <p><a id="checkout-contact-btn" class="btn btn-primary btn-block checkout-page-contact-cta" href="/contact/" style="display: none;">Contact Us</a></p>
+            <p style="font-size:85%;" class="text-muted text-center margin-bottom-none visible-sm-block visible-md-block visible-lg-block">View our <span><a href="/terms/" title="Gruntwork Terms of Service" target="_blank">Terms of Service</a></span></p>
+          </div>
+        </div>
+        <div class="row panel panel-default panel-light">
+          <div class="col-sm-12 panel-body" id="checkout-money-back">
+            <h3 class="no-anchor text-center" style="font-size: 24px">30-Day Money Back Guarantee<a href="/faq/#what-if-i-want-to-cancel-my-gruntwork-subscription" target="_blank"><i class="fa fa-info-circle" data-toggle="tooltip" data-html="true" data-original-title="If you're not happy for any reason, we'll refund 100% of your money within the first 30 days.<br><u>Click the icon for more information</u> <i class='fa fa-caret-down'></i>" style="font-size: 22px;margin-left: 16px;"></i></a></h3>
+          </div>
+        </div>
+      </div>
+      <figure class="grunty-sprite-beta" data-sprite="0">
+        <figcaption class="sr-only">Grunt sprite</figcaption>
+      </figure>
+      <figure id="subscription-type-img" data-subscription-type="">
+        <figcaption class="sr-only">Subscription type image</figcaption>
+      </figure>
+    </div>
+  </div>
+</div>
+
+<div class="panel panel-default panel-light checkout-help-notification">
+  <div class="panel-body text-center">
+    <h2 class="need-help-text">Need help deciding?</h2>
+    <a class="btn btn-primary-hollow btn-sm need-help-button contact-cta checkout-page-contact-cta" href="/contact/">Contact Sales</a>
+  </div>
+</div>
+
+<script src="https://js.chargebee.com/v2/chargebee.js" data-cb-site="gruntwork"></script>
+<script type="text/javascript">
+  window.onload = function() {
+    var cbInstance = Chargebee.getInstance();
```

Comment:
Any reason this is inline like this?

Reply:
No actual reason save for that this checkout-beta page is a duplicate of the original checkout page(except where the new UI changes are made) and the original one had it inline. Will fix that.

Reply:
Can't find any reason for it being inline in the checkout js code which creates instances on demand. I have removed this from both versions.